lib/components/overlays/power_overlay.dart

```diff
@@ -93,62 +92,65 @@ class _PowerOverlayState extends State<PowerOverlay>
                     child: Column(
                       crossAxisAlignment: CrossAxisAlignment.start,
                       children: [
+                        SizedBox(
+                          width: 328,
+                          child: DecoratedBox(
+                            decoration: BoxDecoration(
+                              color: ZenitTheme.of(context).cardColor,
+                            ),
+                            child: Padding(
+                              padding: const EdgeInsets.all(24.0),
+                              child: Column(
+                                crossAxisAlignment: CrossAxisAlignment.start,
+                                children: [
+                                  Text(
+                                    strings.powerOverlay.title,
+                                    style: const TextStyle(
+                                      fontSize: 24,
+                                      fontWeight: FontWeight.w600,
+                                    ),
+                                  ),
+                                  const SizedBox(height: 12),
+                                  Text(strings.powerOverlay.subtitle),
+                                ],
+                              ),
+                            ),
+                          ),
+                        ),
+                        const Spacer(),
                         Padding(
-                          padding: const EdgeInsets.all(24.0),
+                          padding: const EdgeInsets.all(16.0),
                           child: Column(
-                            crossAxisAlignment: CrossAxisAlignment.start,
                             children: [
-                              Text(
-                                strings.powerOverlay.title,
-                                style: const TextStyle(
-                                  fontSize: 24,
-                                  fontWeight: FontWeight.w600,
-                                ),
+                              _powerMenuButton(
+                                strings.powerOverlay.poweroff,
+                                Icons.power_settings_new_rounded,
+                                context,
+                                onPressed: () => ActionManager.powerOff(),
+                              ),
+                              _powerMenuButton(
+                                strings.powerOverlay.sleep,
+                                Icons.brightness_4_outlined,
+                                context,
+                                onPressed: () => ActionManager.suspend(),
+                              ),
+                              _powerMenuButton(
+                                strings.powerOverlay.restart,
+                                Icons.replay_rounded,
+                                context,
+                                onPressed: () => ActionManager.reboot(),
```

Comment:
Remove the () => and () on the lambdas, for example instead of onPressed: () => ActionManager.reboot() use onPressed: ActionManager.reboot, also consider making _powerMenuButton its own widget